.kitchen.yml

```diff
@@ -13,7 +13,7 @@ client_rb:
 platforms:
 - name: centos-6.8
 - name: centos-7.3
-- name: debian-8.6
+- name: debian-8.7
```

Comment:
These can be reduced to centos-6, centos-7,debian-8 etc so you don't need to update the platform each time it comes out.